Source/WebKit/UIProcess/mac/WebContextMenuProxyMac.mm

```diff
@@ -480,14 +480,18 @@ static void updateMenuItemImage(NSMenuItem *menuItem, const WebContextMenuItemDa
 
     if (!hitTestData.absoluteLinkURL.isEmpty()) {
         auto absoluteLinkURL = URL({ }, hitTestData.absoluteLinkURL);
-        if (!absoluteLinkURL.isEmpty())
-            [items addObject:(NSURL *)absoluteLinkURL];
+        if (!absoluteLinkURL.isEmpty()) {
+            if (RetainPtr url = (NSURL *)absoluteLinkURL)
```

Comment:
should this use dynamic_objc_cast (whatever the cast function name is called) instead?

Reply:
This isn't an objc_cast, it's converting a WTF::URL.

Reply:
Looks like the better way to do this is to use `createNSURL()`, so will update to use that.